client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -34,11 +34,18 @@ const DomainRegistrationSuggestion = React.createClass( {
 
 	componentDidMount() {
 		if ( this.props.railcarId && isNumber( this.props.uiPosition ) ) {
+			let resultSuffix = '';
+			if ( this.props.suggestion.isRecommended ) {
+				resultSuffix = 'recommended';
+			} else if ( this.props.suggestion.isBestAlternative ) {
+				resultSuffix = 'best-alternative';
+			}
+
 			tracks.recordEvent( 'calypso_traintracks_render', {
 				railcar: this.props.railcarId,
 				ui_position: this.props.uiPosition,
 				fetch_algo: this.props.fetchAlgo,
-				rec_result: this.props.suggestion.domain_name,
+				rec_result: `${ this.props.suggestion.domain_name }#${ resultSuffix }`,
```

Comment:
This causes `rec_result` to have `#` after the domain name always - even if there's no flag. So it becomes `example.com#` or if there's a flag: `example.com#recommended`.
@sirino, maybe we could just log the flag as a separate field? Suffixing it here seems like it's more limiting (we have to parse it for every result) than keeping it in a separate field (which could then be used to more easily filter the results?). Or am I understanding this incorrectly and this format actually does work for you better? :)